docs/user/technical-reference/04-30-storage-configuration.md

```diff
@@ -0,0 +1,97 @@
+# Registry Storage Configuration
+
+The DockerRegistry CR allows you to store images in three ways: filesystem, Azure, and s3. This document describes how to configure DockerRegistry CR to cooperate with all these storage types.
+
+## Filesystem
+
+The filesystem storage is a built-in storage type based on the PersistentVolumeClaim CR, which is part of the Kubernetes functionality. This is a default DockerRegistry CR configuration, and no additional configuration is needed.
+
+All images pushed to this storage are removed when the Docker Registry is uninstalled or the cluster is removed. Stored images can't be shared between clusters.
+
+### Sample CR
+
+```yaml
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: DockerRegistry
+metadata:
+    name: default
+    namespace: kyma-system
+spec: {}
+```
+
+## Azure
+
+The Azure storage can be configured in the DockerRegistry `spec.storage.azure` field. The only thing that is required is the `secretName` field that needs to contain the name of the Secret with Azure configuration located in the same namespace. The following Secret must have three values inside:
+
+* `container` - contains the name of the storage container
+* `accountKey` - contains the key used to authenticate to the Azure Storage
+* `accountName` - contains the name used to authenticate to the Azure Storage
+
+The images can be stored centrally and shared between clusters so that different registries can reuse specific layers or whole images. After deleting cluster or uninstalling the registry module images will not be removed.
```

Comment:
```suggestion
The images can be stored centrally and shared between clusters so that different registries can reuse specific layers or whole images. After deleting the cluster or uninstalling the registry module, images will not be removed.
```